### PR TITLE
✨ Adding liveramp cmp to registered cmps

### PR DESCRIPTION
**Pull request by nielsbaarsma**
To add LiveRamp to the registered CMPs list.
Added documentation:
- Instructions
- Example
- Example on cmp-vendors page.

LiveRamp Privacy Manager is a consent management solution that includes support for the IAB Europe Transparency & Consent Framework v2. The purpose is to provide an AMP support with the <amp-consent> component by registering in "Supported Consent Management Platforms".

Additional context
We would like to register LiveRamp Privacy Manager as a Supported Consent Management Platform for visibility and ease of implementation. Context: https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/integrating-consent.md

cc @ampproject/wg-monetization

### CODE REVIEW

File: examples/amp-consent/cmp-vendors.amp.html [hunk 1]
```diff
@@ -99,6 +99,7 @@
           <option>ConsentManager</option>
           <option>didomi</option>
           <option>iubenda</option>
+          <option>LiveRamp</option>
```
Comment (micajuine-ho) [inline examples/amp-consent/cmp-vendors.amp.html:102]: nit: Could you do us a favor and move `sirdata` above `opencmp`?

File: examples/amp-consent/cmp-vendors.amp.html [hunk 2]
```diff
@@ -356,6 +357,29 @@ <h3>Image that is NOT blocked by consent</h3>
       </amp-consent>
       <!-- End Usercentrics example -->
 
+      <!-- LiveRamp example -->
+      <amp-consent id="consent" layout="nodisplay" type="LiveRamp">
+        <script type="application/json">
+          {
+            "postPromptUI": "post-consent-ui",
+            "clientConfig": {
+              "appId": "c65a09d5-eda1-4eef-bf38-90e73059f4d6"
```
Comment (micajuine-ho) [inline examples/amp-consent/cmp-vendors.amp.html:366]: nit: place this `amp-consent type=LiveRamp` example below the `iubenda` example

**Review by micajuine-ho — COMMENTED** [review]
@nielsbaarsma Thank you for your contribution! 

Everything looks good, expect for a few nits!

**Comment by nielsbaarsma** [comment]
Hi @micajuine-ho ,

I've pushed the changes you suggested.
Thanks

**Review by micajuine-ho — APPROVED** [review]
Seems like there are some Prettify issues; can you run `gulp prettify --local_changes --fix` and then push the changes?

**Pull request merged**